File: dayTwo/JavaScript/levelTwo/taskFive.js
const prompt = require("prompt-sync")();

let num1 = Number(prompt("Enter number: "));
let num2 = Number(prompt("Enter number: "));
let num3 = Number(prompt("Enter number: "));

largest = num1;

if (num2 > largest){
     console.log(`${num2} is the largest`);
}else if (num3 > largest){
     console.log(`${num3} is the largest`);
}else {
    console.log(`{largest} is the largest`);
}

File: dayTwo/python/levelTwo/task_five.py
num1  = int(input("Enter number 1: "))
num2  = int(input("Enter number 2: "))
num3  = int(input("Enter number 3: "))

largest = num1

if num2 > largest:
    largest = num2
    print(largest)
elif num3 > largest:
    largest = num3
    print(largest)
else: print(largest)
